### PR TITLE
Add missing modes for RouteMode enum

### PR DESCRIPTION
**Pull request by eifinger**
All possible values taken from the documentation:

https://developer.here.com/documentation/routing/topics/resource-param-type-routing-mode.html

### CODE REVIEW

File: herepy/here_enum.py [hunk 1]
```diff
@@ -6,15 +6,19 @@ class RouteMode(Enum):
...
 
     fastest = 'fastest'
+    shortest = 'shortest'
+    balanced = 'balanced'
     bicycle = 'bicycle'
     car = 'car'
+    car_hov = 'carHOV'
+    traffic_enabled = 'traffic:enabled'
     traffic_disabled = 'traffic:disabled'
+    traffic_default = 'traffic:default'
```
Comment (abdullahselek) [inline herepy/here_enum.py]: Thanks for adding new types 👍 could you please also add some basic tests on `test_enum.py`.

Comment (eifinger) [reply herepy/here_enum.py]: Done

**Comment by abdullahselek** [comment]
Thanks for your contribution looks good!

**Comment by abdullahselek** [comment]
Do you want me to release a new version?

**Comment by eifinger** [comment]
That would be awesome!

**Comment by abdullahselek** [comment]
`0.6.3.1` just released.

**Pull request merged**